app/models/spree/print_invoice_configuration.rb

```diff
@@ -2,7 +2,7 @@ module Spree
   class PrintInvoiceConfiguration < Preferences::Configuration
 
     preference :print_invoice_next_number, :integer, :default => nil
-    preference :print_invoice_logo_path, :string, :default => Spree::Config[:admin_interface_logo]
+    preference :print_invoice_logo_path, :string, :default => Spree::Config[:admin_interface_logo].end_with?('.svg') ? 'logo/solidus_logo.png' : Spree::Config[:admin_interface_logo]
```

Comment:
What about make the line shorter?

Reply:
Fixed, @mdesantis! WDYT?

Reply:
Much better!

Reply:
But I'd use `if else end`, ternary operator don't look good when splitted on more lines; I'd do something like this:

```ruby
preference :print_invoice_logo_path, :string,
           :default => if Spree::Config[:admin_interface_logo].end_with?('.svg')
                         'logo/solidus_logo.png'
                       else 
                         Spree::Config[:admin_interface_logo]
                       end
```